blog/2025/pcsx2-2.6/index.mdx

```diff
@@ -0,0 +1,606 @@
+- Don't split/resize buffer when texture buffer width is 0, fixes Baldur's Gate regression. ([#13028](https://github.com/PCSX2/pcsx2/pull/13028))
+- Allow full dirty rect on zero age targets. This fixes Kaena graphical issues at the bottom of the screen. ([#13084](https://github.com/PCSX2/pcsx2/pull/13084))
+- Refactor autoflush handling; detect recursive draw on mipmap layer > 0. This fixes Conspiracy - Weapons of Mass Destruction and Warhammer 40,000 Fire Warrior where lights can be seen through walls. ([#13094](https://github.com/PCSX2/pcsx2/pull/13094))
+
+<SliderCompare
+  before={
+    require("./img/comparisons/13094-MassDestruction-before.webp").default
+  }
+  after={require("./img/comparisons/13094-MassDestruction-after.webp").default}
+/>
+
+<SliderCompare
+  before={require("./img/comparisons/13094-Warhammer-before.webp").default}
+  after={require("./img/comparisons/13094-Warhammer-after.webp").default}
+/>
+
+- Ignore output on RC1/2 when the data is not used. This fixes Sakura Setsugekka and Phantom of Inferno rendering issues. ([#13300](https://github.com/PCSX2/pcsx2/pull/13300))
+
+<SliderCompare
+  before={require("./img/comparisons/13300-Sakura-before.webp").default}
+  after={require("./img/comparisons/13300-Sakura-after.webp").default}
+/>
+
+<SliderCompare
+  before={require("./img/comparisons/13300-Phantom-before.webp").default}
+  after={require("./img/comparisons/13300-Phantom-after.webp").default}
+/>
+
+- Fix temp Z and clear detection regressions. Fixes an issue with temporarily offset Z and a clear misdetection. ([#12983](https://github.com/PCSX2/pcsx2/pull/12983))
+- Fix up invalidly selected Tex in RT targets. Fixes Raider Legends hair and Battlefield 2 regressions. ([#13505](https://github.com/PCSX2/pcsx2/pull/13505))
+- Use accurate fog equation. Avoids using the `mulhrsw` instruction, since it rounds instead of truncating which improves fog emulation in software renderer in some games. ([#13171](https://github.com/PCSX2/pcsx2/pull/13171))
+
+<PCSX2PRLink authors="TheLastRar,govanify" prNums="13682,13697,13614">
+  ### The Resizing Saga
+</PCSX2PRLink>
+
+It all started with [#13613](https://github.com/PCSX2/pcsx2/issues/13613). One of our testers, JordanTheToast had recently upgraded to a Radeon RX 9060 XT. He noticed that resizing PCSX2 while in-game causes massive lag/delay during the resize. This ended up turning into a deep dive that confused the team for days. A look into GPU driver bugs, and the way Qt handles window resize events slowly uncovered the problem.
+
+GovanifY [put up a fight (#13614)](https://github.com/PCSX2/pcsx2/pull/13614) by fixing the way Qt handles window resizing in order to make it not fire up multiple resize events per milliseconds (yes, millisecond).
+
+Following [#13614](https://github.com/PCSX2/pcsx2/pull/13614) it was noticed that NVIDIA has lost smooth rescaling, unlike AMD which retained it.
```

Comment:
The fact that we only specify NVIDIA implicitly tells the reader that the other vendors were fine.

```suggestion
Following [#13614](https://github.com/PCSX2/pcsx2/pull/13614), it was noticed that NVIDIA has lost smooth rescaling.
```